lib/src/com/iunknown.dart

```diff
@@ -46,7 +48,12 @@ class IUnknown implements Finalizable {
   ///
   /// If the COM object implements the interface, then it returns a pointer to
   /// that interface after calling `addRef` on it.
-  int queryInterface(Pointer<GUID> riid, Pointer<Pointer> ppvObject) => ptr
+  @Deprecated('Use toInterface instead. This function will be removed in the '
```

Comment:
I think I'd prefer not to deprecate this, but rather guide people towards `toInterface` as a helper function. The reason is consistency: it's an exported COM function (along with `addRef` and `release`), and I think it would be unique to actually remove it. In theory, someone else might build a COM helper library where they want to call it directly.

Reply:
Done